### PR TITLE
DEPR Enforce ewm.vol deprecation

### PR DESCRIPTION
**Pull request by mroeschke**
Introduced in https://github.com/pandas-dev/pandas/pull/39220

### CODE REVIEW

File: doc/source/whatsnew/v2.0.0.rst [hunk 1]
```diff
@@ -161,6 +161,7 @@ Removal of prior version deprecations/changes
 - Removed deprecated :meth:`Timedelta.delta`, :meth:`Timedelta.is_populated`, and :attr:`Timedelta.freq` (:issue:`46430`, :issue:`46476`)
 - Removed the ``numeric_only`` keyword from :meth:`Categorical.min` and :meth:`Categorical.max` in favor of ``skipna`` (:issue:`48821`)
 - Removed :func:`is_extension_type` in favor of :func:`is_extension_array_dtype` (:issue:`29457`)
+- Removed :meth:`core.window.ewm.ExponentialMovingWindow.vol` (:issue:`39220`)
```
Comment (phofl) [inline doc/source/whatsnew/v2.0.0.rst:164]: Technically core is considered private, can we reference this in another way?

Comment (mroeschke) [reply doc/source/whatsnew/v2.0.0.rst]: Sounds good. Removed the sphinx directive too since this method won't be reference-able

**Comment by jbrockmendel** [comment]
looks like unused imports

**Comment by mroeschke** [comment]
> looks like unused imports

Thanks. Cleaned up

**Comment by phofl** [comment]
thx @mroeschke

**Pull request merged**